draft-ietf-masque-connect-udp-listen.md

```diff
@@ -124,12 +124,15 @@ defined in {{compression}}.
 
 This extension leverages context IDs (see {{Section 4 of CONNECT-UDP}}) to
 compress the target IP address and port when encoding datagrams on the wire.
-Endpoint start by registering a context ID and the IP/ports it's associated
+Endpoint starts by registering a context ID and the IP/ports it's associated
 with by sending a COMPRESSION_ASSIGN capsule to its peer. The peer will then
 echo that capsule to indicate it's received it and estabished its own mapping.
 From then on, both endpoints are aware of the context ID and can send
 compressed datagrams. Later, any endpoint can decide to close the compression
-context by sending a COMPRESSION_CLOSE capsule.
+context by sending a COMPRESSION_CLOSE capsule. Only one such active context
```

Comment:
This isn't very clear to me. It could be parsed to mean "only one context is allowed at a time" which right. It could also be parsed as "you can't have two active contexts for the same address/port combo" - is that what we mean? Maybe it's simplest to describe this in terms of capsules. Perhaps `Endpoints MUST NOT send two COMPRESSION_ASSIGN capsules with the same context ID. If a recipient detects a repeated context ID, it MUST consider the capsule as malformed"